clientv3/integration/kv_test.go

```diff
@@ -28,6 +29,53 @@ import (
 	"golang.org/x/net/context"
 )
 
+func assertRpctypesError(t *testing.T, err, expected error) {
+	if err == nil {
+		t.Fatalf("expected %v, got <nil>", expected)
+	}
+	if err != expected {
+		t.Fatalf("expected %v, got %v", expected, err)
+	}
+	if reflect.TypeOf(err) != reflect.TypeOf(expected) {
```

Comment:
will remove. Different types will still be equal in `err != expected` even when we do the actual type-conversion, I think.